### DISCUSSION TITLE
de/1-04-ai-application-areas/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#61] [General]
# de/1-04-ai-application-areas/

https://ai.lernos.org/de/1-04-ai-application-areas/

**Comment by MelanieVirtuelleAssistenz** [comment]
Bei "Text" in erster Tabellenzeile: Email ist falsch geschrieben --> Richtig: E-Mail

**Comment by MelanieVirtuelleAssistenz** [comment]
Erste Zeile in Tabelle: "Text": Email ist falsch geschrieben. Richtig: E-Mail

**Comment by olelai** [comment]
Kategorie: Text
Service-Software, die Kundenanfragen analysiert, kategorisiert und in Ordner sortiert, Kundeninformationen aus ERP- und CRM-System zusammenstellt, Hintergrundinformationen zur Beantwortung der Anfrage zur Verfügung stellt (z.B. aus den firmeneigenen FAQs) und eine Antwort vorschlägt. Die KI kann sogar selbstständig die Antwort verschicken. Solche Software gibt es schon, z.B. Sematell und Zendesk.

**Comment by moklusch** [comment]
Der Artikel schlägt dafür das WINS-Meme vor, mit DEM du prüfen kannst, wie viel Aufwand der eigenen Tätigkeiten auf Umgang mit Texten (Words), Bildern (Images), Zahlen (Numbers) und Audio (Sounds) entfällt. ==> DEM fehlt

**Reply by ThomasJenewein** [reply]
Bei Audio bitte noch Audio to Text  

Transkription von Audio, z.B. für Podcasts für Accessibility & SEO 

Bei Text würde ich noch Übersetzung reinmachen  - das kann alleine oder als Teil der Erstellung gemacht werden (auch bei Video ist Mehrsprachigkeit ein Thema - da ist es schon drinnen) 

Daten sehe ich als eigenen Bereich 

Daten Analysieren und Visualisieren - zB mit Advanced Data Analysis 

Code korrigieren, vergleichen könnte man noch bei Code hinzufügen

**Comment by KathaZ** [comment]
A1/Z2: (AAAI)*,*
A1/Z3: Hugging Face verlinken => https://huggingface.co/
Tabelle, Audio: Voiceover-Stimmen
Tabelle, Audio: oder -*m*usik
